### DISCUSSION TITLE
Surface plasmon resonance using MEEP

### DISCUSSION THREAD

**Discussion by varunsv24** [NanoComp/meep#2577] [Q&A]
How to get surface plasmon resonance of a metallic sphere data using MEEP. Please suggest codes.

**Comment by dalarev** [comment]
Have you seen the related example in the official documentation?

[https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/#mie-scattering-of-a-lossless-dielectric-sphere](https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/#mie-scattering-of-a-lossless-dielectric-sphere)

Regarding the metallic component, see [this](https://github.com/NanoComp/meep/discussions/2218#discussioncomment-3512841) comment for some guidance.
